adrs/Authors - Books API.md

```diff
@@ -0,0 +1,31 @@
+# How to store book authors in the database?
+
+## Status
+Proposed
+
+## Context
+We have a book and we have an author/authors of this book. How should we store information about author/authors?
+
+## Alternatives
+### One alternative
+As string in the “Book” table - store fullnames as a string array.
+#### Pros
+- There will be no need to implement cascading deletion/editing when deleting/editing a book;
+- Few people write more than 1 book from technical authors, a dictionary with authors **for our** library will not be reused often.
+#### Cons
+- If necessary, it will be more difficult to put the authors in a separate table;
+- It will be more difficult to search for books by author;
+- If there is a mistake in the author's name, you will have to correct it in each book separately.
+### Two alternative
+As a different table named “Author” with M:M relation to “Book” table.
```

Comment:
I suggest if it's not too hard to add here mermaid-diagrams with the piece of base you're talking about here, so that the variant is immediately demonstrative.